js/content.js

```diff
@@ -8,11 +8,7 @@ const bridgeEvents = {
 };
 let port = null;
 
-browser.runtime.onMessage.addListener((msg, sender, sendResponse) => {
-    if (msg.text === 'loadster_content_script_loaded') {
-        sendResponse({ status: true });
-    }
-});
+browser.runtime.onMessage.addListener((msg, sender) => new Promise((resolve) => resolve(true)));
```

Comment:
I guess we didn't care if it was `loadster_content_script_loaded` since we're just responding true?

Reply:
yeah, that can be any message, we are only interested in the fact of messaging. And we also don't care about other apps conflicts here...  thinking twice, maybe I should create some extension with some nasty events to stress test loadster-extension. Thank you for highlighting that